### DISCUSSION TITLE
i2pd.xyz vs i2pd.website

### DISCUSSION THREAD

**Discussion by anikey-from-i2p** [PurpleI2P/i2pd#2067] [Q&A] [answered]
Posting this in discussion, because it's not related to the software itself.

Which one should people use and link to?

Also, both of them link to "i2p.net" instead of "geti2p.net".

**Comment by r4sas** [comment, marked as answer]
i2pd.website is main project address, while i2pd.xyz is mirror and used for packages repository.
